js/packages/openinference-core/src/trace/trace_config/types.ts

```diff
@@ -0,0 +1,43 @@
+import { AttributeValue } from "@opentelemetry/api";
+import { OISpan } from "./OISpan";
+
+export type TraceConfigOptions = {
+  hideInputs?: boolean;
+  hideOutputs?: boolean;
+  hideInputMessages?: boolean;
+  hideOutputMessages?: boolean;
+  hideInputImages?: boolean;
+  hideInputText?: boolean;
+  hideOutputText?: boolean;
+  hideEmbeddingVectors?: boolean;
+  base64ImageMaxLength?: number;
+};
+
+export type TraceConfig = Readonly<Required<TraceConfigOptions>>;
+
+export type TraceConfigKey = keyof TraceConfig;
+
+export type TraceConfigMetadata =
```

Comment:
this type name is kinda confusing - it's like the TraceConfigFlag or something like that

Reply:
good idea